src/Microsoft.ML.Data/Scorers/BinaryClassifierScorer.cs

```diff
@@ -188,7 +188,7 @@ private protected override void SaveAsOnnxCore(OnnxContext ctx)
             Host.Assert(Bindings.InfoCount >= 2);
 
             if (!ctx.ContainsColumn(DefaultColumnNames.Features))
-                return;
+                throw Contracts.ExceptParam(nameof(DefaultColumnNames.Features), $"Please use default Column Names: '{DefaultColumnNames.Features}'");
```

Comment:
could you please add corresponding test to verify this logic?

Reply:
yep